CONTRIBUTING.rst

```diff
@@ -79,9 +79,21 @@ Development Workflow
          PYTHONPATH=. pytest networkx
 
    * Finally, we recommend you use a pre-commit hook, which runs black when
-     you type ``git commit``::
+     you type ``git commit``. To do this, you need to have the pre-commit package manager installed, followed by the setup of the git hook scripts ::
```

Comment:
Explicitly installing `pre-commit` shouldn't be necessary as it's included in `requirements/developer.txt` which is used in the previous set of instructions.

Reply:
Okay sir, please how can I adjust it in a way that it will not be too explicit and also in a way which will help newcomers who land on the contribution page.